materials/version-control-R.md

```diff
@@ -274,16 +252,8 @@ add_size_classes <- function(df) {
 > Have students email a link to their repo to their instructor once they have
 > finished Pushing Changes
 >
-> The instructor should then commit the following code to their repo
->
-> get_size_class <- function(ear_length){
->   # Calculate the size class for one or more ear lengths
->   ear_lengths <- ifelse(ear_length > 10, "large", "small")
->   return(ear_lengths)
-> }
->
-> With the commit message:
-> Add function for determining ear length class
+> The instructor should then commit the *Add output table* code to their repo
```

Comment:
I found this section confusing also while I was making my revisions. Here's what I'm thinking now that I have some clarity:

* The "Push to a Remote" lecture section should not include any new code. It will just be a demo of pushing the commits from earlier in the lecture. 
* Students can then do (or not) the "Pushing Changes" exercise, which adds the GC-content function.
* The "Pulling and Pushing" exercise will then use the *Add output table* code chunk. This could be added as another section to the lecture if you want students to follow along with the instructor.